code/modules/client/preference/preferences.dm

```diff
@@ -133,6 +133,7 @@ GLOBAL_LIST_INIT(special_role_times, list( //minimum age (in days) for accounts
 	var/body_accessory = null
 
 	var/speciesprefs = 0//I hate having to do this, I really do (Using this for oldvox code, making names universal I guess
+	var/r_company = "Morpheus Cyberkinetics"
```

Comment:
I had considered adding `r_company` to the SQL. However, I do not think it is worth it. I also want feedback here. 

@blinkdog I am not sure this spot has to show the selected model loadout. A player could then completely change their characters' prostheses and this would never update. In a prototype version, I had this as "Click Here". I am thinking of changing this to "Options".

What do you think?